lms/djangoapps/bulk_email/models.py

```diff
@@ -215,6 +216,10 @@ def render_htmltext(self, htmltext, context):
         Convert HTML text body (`htmltext`) into HTML email message using the
         stored HTML template and the provided `context` dict.
         """
+        # HTML-escape string values in the context (used for keyword substitution).
+        for key, value in context.iteritems():
+            if isinstance(value, basestring):
```

Comment:
Yes, it will. str and unicode extend from basestring.

http://stackoverflow.com/questions/1979004/what-is-the-difference-between-isinstanceaaa-basestring-and-isinstanceaaa